figures/time/example_lomb_scargle.py

```diff
@@ -0,0 +1,17 @@
+import numpy as np
+from astropy.table import Table
+import sys
+
+from .....gammapy.gammapy.time.lomb_scargle import lomb_scargle
```

Comment:
You cannot import like this from scripts outside the Gammapy package.
It should be:
```
from gammapy.time.lomb_scargle import lomb_scargle
```
and then your Python path has to be configured to find the right version of Gammapy.
To check your Python path:
```
import sys
print(sys.path)
```